### DISCUSSION TITLE
I cannnot read document pages

### DISCUSSION THREAD

**Discussion by takakikasuga** [edmundhung/conform#808] [Q&A]
Hi!!  @edmundhung 

https://conform.guide/

now,,,
I have been to the page, but 404 not found was shown...

![スクリーンショット 2024-10-27 9 08 56](https://github.com/user-attachments/assets/7a47ee58-f972-45f2-b1b1-3404c8d705c2)

could you fix this problem...🙏

**Comment by edmundhung** [comment]
It should be up now. I didn't notice the github token is just expired. Thanks for reporting!

**Comment by takakikasuga** [comment]
@edmundhung 

I just have looked at your doument page!!
thank you!!
